<Type Name="ForeignKeyReferenceAlreadyHasValueException" FullName="System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="d8770cffaeaf250a5417fb744564f86f9759ee9f" />
    <Meta Name="ms.sourcegitcommit" Value="6a0b904069161bbaec4ffd02aa7d9cf38c61e72e" />
    <Meta Name="ms.translationtype" Value="MT" />
    <Meta Name="ms.contentlocale" Value="pl-PL" />
    <Meta Name="ms.lasthandoff" Value="06/24/2018" />
    <Meta Name="ms.locfileid" Value="36689304" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class ForeignKeyReferenceAlreadyHasValueException : InvalidOperationException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit ForeignKeyReferenceAlreadyHasValueException extends System.InvalidOperationException" />
  <TypeSignature Language="DocId" Value="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" />
  <TypeSignature Language="VB.NET" Value="Public Class ForeignKeyReferenceAlreadyHasValueException&#xA;Inherits InvalidOperationException" />
  <TypeSignature Language="C++ CLI" Value="public ref class ForeignKeyReferenceAlreadyHasValueException : InvalidOperationException" />
  <TypeSignature Language="F#" Value="type ForeignKeyReferenceAlreadyHasValueException = class&#xA;    inherit InvalidOperationException" />
  <AssemblyInfo>
    <AssemblyName>System.Data.Linq</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.InvalidOperationException</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
      <span data-ttu-id="520ef-101">Reprezentuje błędów występujących podczas próby zmiany klucza obcego, gdy obiekt jest już załadowany.</span>
      <span class="sxs-lookup">
        <span data-stu-id="520ef-101">Represents errors that occur when an attempt is made to change a foreign key when the entity is already loaded.</span>
      </span>
    </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="520ef-102">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> klasy.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-102">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ForeignKeyReferenceAlreadyHasValueException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ForeignKeyReferenceAlreadyHasValueException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="520ef-103">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> używając dostarczany przez system komunikatu opisującego błąd.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-103">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class with a system-supplied message that describes the error.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="520ef-104">Ten konstruktor inicjuje <xref:System.Exception.Message%2A> właściwość nowego wystąpienia dostarczony przez system komunikat, który opisuje błąd, takim jak "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply".</span><span class="sxs-lookup"><span data-stu-id="520ef-104">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as "DefaultMessageDisplayedByParameterlessCtorWriterMustSupply".</span></span> <span data-ttu-id="520ef-105">Ten komunikat uwzględnia bieżącą kulturę systemu.</span><span class="sxs-lookup"><span data-stu-id="520ef-105">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="520ef-106">W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.InsufficientMemoryException>.</span><span class="sxs-lookup"><span data-stu-id="520ef-106">The following table shows the initial property values for an instance of <xref:System.InsufficientMemoryException>.</span></span>  
  
|<span data-ttu-id="520ef-107">Właściwość</span><span class="sxs-lookup"><span data-stu-id="520ef-107">Property</span></span>|<span data-ttu-id="520ef-108">Wartość</span><span class="sxs-lookup"><span data-stu-id="520ef-108">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="520ef-109">`null`.</span><span class="sxs-lookup"><span data-stu-id="520ef-109">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="520ef-110">Zlokalizowany ciąg komunikatu o błędzie.</span><span class="sxs-lookup"><span data-stu-id="520ef-110">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ForeignKeyReferenceAlreadyHasValueException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ForeignKeyReferenceAlreadyHasValueException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException : string -&gt; System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" Usage="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="520ef-111">Komunikat, który opisuje wyjątek.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-111">The message that describes the exception.</span>
          </span>
          <span data-ttu-id="520ef-112">Obiekt wywołujący tego konstruktora jest wymagany w celu zagwarantowania, że ten ciąg został zlokalizowany w bieżącej kulturze systemu.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-112">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="520ef-113">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> klasy z określony komunikat zawierający opis błędu.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-113">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class with a specified message that describes the error.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="520ef-114">W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span><span class="sxs-lookup"><span data-stu-id="520ef-114">The following table shows the initial property values for an instance of <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span></span>  
  
|<span data-ttu-id="520ef-115">Właściwość</span><span class="sxs-lookup"><span data-stu-id="520ef-115">Property</span></span>|<span data-ttu-id="520ef-116">Wartość</span><span class="sxs-lookup"><span data-stu-id="520ef-116">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="520ef-117">`null`.</span><span class="sxs-lookup"><span data-stu-id="520ef-117">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="520ef-118">Ciąg z komunikatem o określonych w `message`.</span><span class="sxs-lookup"><span data-stu-id="520ef-118">The error message string specified in `message`.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public ForeignKeyReferenceAlreadyHasValueException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; ForeignKeyReferenceAlreadyHasValueException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException : string * Exception -&gt; System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" Usage="new System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.Data.Linq</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="520ef-119">Komunikat, który opisuje wyjątek.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-119">The message that describes the exception.</span>
          </span>
          <span data-ttu-id="520ef-120">Obiekt wywołujący tego konstruktora jest wymagany w celu zagwarantowania, że ten ciąg został zlokalizowany w bieżącej kulturze systemu.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-120">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="520ef-121">Wyjątek, który jest przyczyną bieżącego wyjątku.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-121">The exception that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="520ef-122">Jeśli <c>innerException</c> parametr nie jest <see langword="null" />, bieżący wyjątek jest zgłaszany w <see langword="catch" /> bloku, który obsługuje wyjątek wewnętrzny.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-122">If the <c>innerException</c> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="520ef-123">Inicjuje nowe wystąpienie klasy <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> z określonego komunikatu o błędzie i odwołania do wyjątek wewnętrzny będący przyczyną tego wyjątku.</span>
          <span class="sxs-lookup">
            <span data-stu-id="520ef-123">Initializes a new instance of the <see cref="T:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="520ef-124">Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A> właściwości.</span><span class="sxs-lookup"><span data-stu-id="520ef-124">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="520ef-125"><xref:System.Exception.InnerException%2A> Właściwość zwraca tę samą wartość, która została przekazana do konstruktora, lub `null` Jeśli <xref:System.Exception.InnerException%2A> właściwości nie dostarcza do konstruktora wartości wyjątku wewnętrznego.</span><span class="sxs-lookup"><span data-stu-id="520ef-125">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="520ef-126">W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span><span class="sxs-lookup"><span data-stu-id="520ef-126">The following table shows the initial property values for an instance of <xref:System.Data.Linq.ForeignKeyReferenceAlreadyHasValueException>.</span></span>  
  
|<span data-ttu-id="520ef-127">Właściwość</span><span class="sxs-lookup"><span data-stu-id="520ef-127">Property</span></span>|<span data-ttu-id="520ef-128">Wartość</span><span class="sxs-lookup"><span data-stu-id="520ef-128">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="520ef-129">`null`.</span><span class="sxs-lookup"><span data-stu-id="520ef-129">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="520ef-130">Ciąg z komunikatem o określonych w `message`.</span><span class="sxs-lookup"><span data-stu-id="520ef-130">The error message string specified in `message`.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>